docs/tour/adding-capabilities.mdx

```diff
@@ -381,6 +452,76 @@ func (h HttpServer) Handle(request HttpRequest, responseWriter HttpResponseWrite
 	okResponse := http.Ok[HttpOutgoingResponse, HttpError](httpResponse)
 	http.StaticResponseOutparamSet(responseWriter, okResponse)
 }
+```
+
+  </TabItem>
+  <TabItem value="typescript" label="TypeScript Component">
+
+  In your template, we already included the `wasi:keyvalue` interface for interacting with a key value store. We can also use the `wasi:logging` interface to log the name of each person we greet. Before you can use the functionality of those interfaces, you'll need to add a few imports to your `wit/hello.wit` file:
+```wit
+package wasmcloud:hello;
+
+world hello {
+  // highlight-start
+  import wasi:keyvalue/atomic;
+  import wasi:keyvalue/readwrite;
+  import wasi:logging/logging;
+  // highlight-end
+
+  export wasi:http/incoming-handler@0.2.0-rc-2023-12-05;
+}
+```
+
+```typescript
+import {
+  IncomingRequest,
+  ResponseOutparam,
+  OutgoingResponse,
+  Fields,
+} from "wasi:http/types@0.2.0-rc-2023-12-05";
+
+//highlight-start
+//@ts-expect-error
```

Comment:
```suggestion
//@ts-expect-error -- some wasi modules do not have correct types yet but can still be imported
```

Now that I think about it, it might be worth adding a note here because `@ts-expect-error` it is a codesmell

Reply:
Aw, I was hoping you wouldn't smell it!